web/.env.development

```diff
@@ -1,3 +1,4 @@
+VITE_APP_DANDI_BACKEND_ROOT=http://localhost:8000
 VITE_APP_DANDI_API_ROOT=http://localhost:8000/api/
 VITE_APP_OAUTH_API_ROOT=http://localhost:8000/oauth/
```

Comment:
I think these can be refactored now to inject the `DANDI` and `API` variables into the config at runtime, but it seemed like it was out of scope for this PR.